### PR TITLE
task 9374 + 9375: confirm send transtion screen - handle event + fetch data, api

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9374
https://edu-redmine.sun-asterisk.vn/issues/9375
### Goals :soccer:

Task - 9374
- Click Send button, an alert displays and require user enter password to complete the transaction if user enter correct password, it redirect to Asset Screen, add a new pedding transaction to transaction tableview on Asset Screen
- Click setting nav button to go to Advanced Transaction Screen
Task -9375
Fetch data from API:
- Get recomend gas price and gas limit for transaction
### Implementation Details :construction:

https://gfycat.com/RewardingCornyIndianelephant

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

File: CryptoWallet/Scenes/Home/Wallet/WalletViewController.swift [hunk 1]
```diff
@@ -193,45 +193,41 @@ final class WalletViewController: UIViewController {
...
-        group.enter()
         assetRespository.getAssetList(address: address) { (result) in
             switch result {
             case .success(let tokenListResponse):
                 guard let tokenListResponse = tokenListResponse else {
-                    group.leave()
                     return
                 }
                 self.allAssetList[0].amount = tokenListResponse.etherBalance
+                self.assetRespository.getEthereumInfo { (result) in
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Home/Wallet/WalletViewController.swift]: weak self

File: CryptoWallet/Scenes/Home/Wallet/WalletViewController.swift [hunk 2]
```diff
@@ -193,45 +193,41 @@ final class WalletViewController: UIViewController {
...
+                    switch result {
+                    case .success(let ethereumMarketResponse):
+                        if let fetchedEthereum = ethereumMarketResponse {
+                            self.allAssetList[0].name = fetchedEthereum.name
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Home/Wallet/WalletViewController.swift]: AssetInfo()

**Comment by vanpd-0859** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**